docs/standard/serverless-architecture/serverless-business-scenarios/index.md

```diff
@@ -0,0 +1,97 @@
+
+## Parse a CSV file and insert into a database
+
+Extract, Transform, and Load (ETL) is a common business function that is often necessary for integration between different systems. Traditional approaches often involve setting up dedicated FTP servers then deploying scheduled jobs to parse files and translate them for business use. Serverless architecture makes the job easier because a trigger can fire when the file is uploaded. Azure Functions is composed of small pieces of code that focus on a specific problem and are ideal for tackling tasks like ETL.
+
+![ETL architecture](./media/csvimport.png)
+
+For source code and a hands-on lab, see: [CSV import lab](https://github.com/JeremyLikness/azure-fn-file-process-hol).
+
+## Shorten links and track metrics
+
+Link shortening was originally used to encode URLs in short twitter posts to accommodate the 140 character limit. They've grown to encompass several uses, most commonly to track click-throughs for analytics. The link shortener scenario is an entirely serverless application for managing links and reporting metrics.
+
+Azure Functions is used to serve a Single Page Application (SPA) that allows you to paste the long URL and generate short URLs. The URLs are tagged to track things like campaigns (topics) and mediums (such as social networks that the links are posted to). The short code is stored in Azure Table Storage as the key, with the long URL as the value. When you click on the short link, another function looks up the long URL, sends a redirect, and places information about the event on a queue. Another Azure Function processes the queue and places the information into CosmosDB.
+
+![Link shortener architecture](./media/link-shortener-architecture.png)
+
+You can then create a PowerBI dashboard to gather insights about the data collected. On the backend, Application Insights provide important metrics. Telemetry includes how long it takes for the average user to redirect and how long it takes to access Azure Table Storage.
+
+![Power BI example](./media/power-bi-example.png)
+
+The full link shortener repository with instructions is available here: [Serverless URL shortener](https://github.com/jeremylikness/serverless-url-shortener). You can read about a simplified version here: [Azure Storage for serverless .NET apps in minutes](https://blogs.msdn.microsoft.com/webdev/2018/01/25/azure-storage-for-serverless-net-apps-in-minutes/).
+
+## Verify device connectivity using a ping
+
+The sample consists of an Azure IoT Hub and an Azure Function. The Azure Function is triggered when a new message is received on the IoT Hub. The serverless code sends the same message content back to the device that sent it. The project has all the code and deployment configuration needed for the solution.
```

Comment:
2nd sentence is passive voice. Consider "A new message on the IoT Hub triggers and Azure Function."